runtime-modules/content-directory/src/helpers.rs

```diff
@@ -0,0 +1,220 @@
+use crate::*;
+use core::ops::{Deref, DerefMut};
+
+/// Wrapper for existing `PropertyValue` and its respective `Class` `Property`
+pub struct ValueForExistingProperty<'a, T: Trait>(&'a Property<T>, &'a PropertyValue<T>);
+
+impl<'a, T: Trait> ValueForExistingProperty<'a, T> {
+    /// Create single instance of `ValueForExistingProperty` from provided `property` and `value`
+    fn new(property: &'a Property<T>, value: &'a PropertyValue<T>) -> Self {
+        Self(property, value)
+    }
+
+    /// Retrieve `Property` reference
+    pub fn get_property(&self) -> &Property<T> {
+        self.0
+    }
+
+    /// Retrieve `Property` and `PropertyValue` references
+    pub fn unzip(&self) -> (&Property<T>, &PropertyValue<T>) {
+        (self.0, self.1)
+    }
+}
+
+/// Mapping, used to represent `PropertyId` relation to its respective `ValueForExistingProperty` structure
+pub struct ValuesForExistingProperties<'a, T: Trait>(
```

Comment:
Because i wanted to encapsulate its https://github.com/iorveth/substrate-runtime-joystream/blob/update_entity_permissions_extrinsic/runtime-modules/content-directory/src/helpers.rs#L51 associated method. It's not possible to implement methods for type aliases. We also can create a specific trait for `ValuesForExistingProperties`, though i did not see any advantage.